### PR TITLE
add redeem info to other transactions on portfolio

### LINKED ISSUE #1324: Display reedeem info on transaction history

### PR DESCRIPTION
**Pull request by TvrtkoM**
close #1324

### CODE REVIEW

File: lib/util/merge-sorted-arrays.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import PriorityQueue from "js-priority-queue";
+
+interface Entry<T> {
+  array: T[];
+  index: number;
+}
+
+function mergeSortedArrays<
```
Comment (Robiquet) [inline lib/util/merge-sorted-arrays.ts:8]: Why can't we just merge then sort by block number after?

Comment (TvrtkoM) [reply lib/util/merge-sorted-arrays.ts:8]: this is more efficient on large datasets

Comment (Robiquet) [reply lib/util/merge-sorted-arrays.ts:8]: How much time does this save for a typical user? Seems a bit overkill to add a new library for this. I am a bit skeptical or whether this is actually more performant as it's still looping over both of the arrays

Comment (TvrtkoM) [reply lib/util/merge-sorted-arrays.ts:8]: image

Comment (TvrtkoM) [reply lib/util/merge-sorted-arrays.ts:8]: image

Comment (TvrtkoM) [reply lib/util/merge-sorted-arrays.ts:8]: image

Comment (Robiquet) [reply lib/util/merge-sorted-arrays.ts:8]: It takes less than 1ms to sort an array of 10000 elements so I don't think this is something we need to optimse
image

File: package.json [hunk 2]
```diff
@@ -100,6 +100,7 @@
     "@testing-library/react": "^14.0.0",
     "@types/flexsearch": "^0.7.2",
     "@types/jest": "^29.5.0",
+    "@types/js-priority-queue": "^0.0.6",
```
Comment (Robiquet) [inline package.json:103]: Think this can be removed too

**Review by Robiquet — APPROVED** [review]

**Pull request merged**